src/test/java/ui/OmeZarrS3V4Opener.java

```diff
@@ -3,41 +3,51 @@
 import bdv.util.BdvFunctions;
 import mpicbg.spim.data.SpimData;
 import org.embl.mobie.io.ome.zarr.openers.OMEZarrS3Opener;
+import org.junit.jupiter.api.Test;
 
 import java.io.IOException;
 
 public class OmeZarrS3V4Opener {
-    public static void main(String[] args) throws IOException {
-        showYX();
-    }
 
-    public static void showYX() throws IOException {
+    @Test
+    public void showYX() throws IOException {
         SpimData image = OMEZarrS3Opener.readURL("https://s3.embl.de/i2k-2020/ngff-example-data/v0.4/yx.ome.zarr");
         BdvFunctions.show(image);
     }
 
-    public static void showZYX() throws IOException {
+    @Test
+    public void showZYX() throws IOException {
         SpimData image = OMEZarrS3Opener.readURL("https://s3.embl.de/i2k-2020/ngff-example-data/v0.4/zyx.ome.zarr");
         BdvFunctions.show(image);
     }
 
-    public static void showCYX() throws IOException {
+    @Test
+    public void showCYX() throws IOException {
         SpimData image = OMEZarrS3Opener.readURL("https://s3.embl.de/i2k-2020/ngff-example-data/v0.4/cyx.ome.zarr");
         BdvFunctions.show(image);
     }
 
-    public static void showTYX() throws IOException {
+    @Test
+    public void showTYX() throws IOException {
         SpimData image = OMEZarrS3Opener.readURL("https://s3.embl.de/i2k-2020/ngff-example-data/v0.4/tyx.ome.zarr");
         BdvFunctions.show(image);
     }
 
-    public static void showTCYX() throws IOException {
+    @Test
+    public void showTCYX() throws IOException {
         SpimData image = OMEZarrS3Opener.readURL("https://s3.embl.de/i2k-2020/ngff-example-data/v0.4/tcyx.ome.zarr");
         BdvFunctions.show(image);
     }
 
-    public static void showTCZYX() throws IOException {
+    @Test
+    public void showTCZYX() throws IOException {
         SpimData image = OMEZarrS3Opener.readURL("https://s3.embl.de/i2k-2020/ngff-example-data/v0.4/tczyx.ome.zarr");
         BdvFunctions.show(image);
     }
+
+    @Test
+    public void multiImg() throws IOException {
```

Comment:
> It would be good to add a test for https://s3.embl.de/i2k-2020/ngff-example-data/v0.4/multi-image.ome.zarr (I have fixed the data there) and then this is good to go from my side.

Here it is. :)

Reply:
@KateMoreva We should add there code that checks that the correct number of Sources is loaded. Do you know how to do this? Otherwise I can add it.